src/App.jsx

```diff
@@ -18,9 +20,31 @@ function App() {
           />
           <Route
             exact
-            path="/game/:gameId"
+            path="/category/:catId"
             render={(routeProps) => (
-              <GameForum id={routeProps.match.params.gameId} {...routeProps} />
+              <Home
+                category
```

Comment:
I decided to just do this rather than have components looking at routeprops themselves to decide what to render. 

This works for now but looking at the three different versions of Home being rendered by these routes makes me feel like we might have to just separate them into three different components at some stage when they start to have more/different features.